packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -93,14 +88,13 @@ export const defaultState: AppState = {
   connectionState: ConnectionState.LOADING,
   l1NetworkChainId: null,
   l2NetworkChainId: null,
-  verifying: WhiteListState.ALLOWED,
   selectedToken: null,
   isDepositMode: true,
   sortedTransactions: derived((s: AppState) => {
     const transactions = s.arbTokenBridge?.transactions?.transactions || []
     return filterTransactions(
       [...transactions],
-      s.arbTokenBridge.walletAddress,
+      getAccount().address || '',
```

Comment:
Instead of passing `''` we should have address check above this return and do an early return if we don't detect a proper address.

Reply:
good point, it's cleaner